test/db/cmd/dwarf

```diff
@@ -1797,3 +1797,1265 @@ EXPECT=<<EOF
 |       |   0x004019ce      e82d280200     call  sym.fpc_get_output
 EOF
 RUN
+
+NAME=Invalid kind for DW_AT_name
+FILE=bins/elf/dwarf3_c.elf.patched0
+CMDS=<<EOF
+aaa
+id
+EOF
+EXPECT=<<EOF
+   1      DW_TAG_compile_unit       [has children] (0x0)
+    DW_AT_producer                 DW_FORM_strp                  
+    DW_AT_language                 DW_FORM_data1                 
+    DW_AT_name                     DW_FORM_strp                  
+    DW_AT_comp_dir                 DW_FORM_strp                  
+    DW_AT_low_pc                   DW_FORM_addr                  
+    DW_AT_high_pc                  DW_FORM_addr                  
+    DW_AT_stmt_list                DW_FORM_data4                 
+   2      DW_TAG_variable           [no children] (0x13)
+    DW_AT_name                     DW_FORM_string                
+    DW_AT_decl_file                DW_FORM_data1                 
+    DW_AT_decl_line                DW_FORM_data1                 
+    DW_AT_decl_column              DW_FORM_data1                 
+    DW_AT_type                     DW_FORM_ref4                  
+    DW_AT_external                 DW_FORM_flag                  
+    DW_AT_location                 DW_FORM_block1                
+   3      DW_TAG_base_type          [no children] (0x26)
+    DW_AT_byte_size                DW_FORM_data1                 
+    DW_AT_encoding                 DW_FORM_data1                 
+    DW_AT_name                     DW_FORM_data4                 
```

Comment:
Binary patched here. Normally `DW_AT_name` would have form `strp` or `string`. `DW_FORM_data4` does not make much sense.
This causes the nonsensical
```
<0x42>: Abbrev Number: 3    (DW_TAG_base_type)
     DW_AT_byte_size           : 4
     DW_AT_encoding            : 5
     DW_AT_name                : 7630441
```
below (note that DW_AT_name has some random integer value now)